pkg/plugins/plugins.go

```diff
@@ -53,10 +54,15 @@ type InitializerPlugin interface {
 // InitializersFromFolder loads initialization plugins compiled as .so files
 // from a folder, runs them on a received pointer to a Service
 func InitializersFromFolder(path string, service *Service) error {
-	return loadPlugins(path, func(fileName string, p plugin.Symbol) error {
-		plug, ok := p.(InitializerPlugin)
+	return loadPlugins(path, func(fileName string, pluginSymbol plugin.Symbol) error {
```

Comment:
This code is already merged on master, and should not be contained in the pull-request.
You should rebase on master instead of actual origin/dev/ybriffa/master